src/index.js

```diff
@@ -2,14 +2,17 @@ import React from 'react';
 import ReactDOM from 'react-dom/client';
 import './index.css';
 import { BrowserRouter } from 'react-router-dom';
+import { Provider } from 'react-redux';
 import App from './App';
+import store from './redux/configureStore';
 
 const root = ReactDOM.createRoot(document.getElementById('root'));
 root.render(
   <React.StrictMode>
     <BrowserRouter>
-      <App />
+      <Provider store={store}>
+        <App />
+      </Provider>
```

Comment:
- [Optional] Your design is almost exceptional @aubinleyoung, But as you can see, on a screen size having a larger height of screen your design seems to remain short on the top. This makes your design a bit uncomfortable from a UI perspective. It would be created to always have the same color on the background to make the design impressive. 
 
![image](https://user-images.githubusercontent.com/69077061/225974047-0de4b1e7-311c-4403-b557-8b18e18d3630.png)